### PR TITLE
New module: Varlociraptor

### LINKED ISSUE #2382: new module: varlociraptor/varlociraptor
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [ ] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by matthdsm**
New module, varlociraptor
closes #2382

- [x] estimate alginment-properties
- [x] preprocess
- [ ] call_generic
- [ ] call_tumornormal
- [x] decodephred
- [ ] genotype

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/varlociraptor/call/main.nf [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+//               where "task.ext" is a string.
+//               Any parameters that need to be evaluated in the context of a particular sample
+//               e.g. single-end/paired-end data MUST also be defined and evaluated appropriately.
+// TODO nf-core: Software that can be piped together SHOULD be added to separate module files
+//               unless there is a run-time, storage advantage in implementing in this way
+//               e.g. it's ok to have a single module for bwa to output BAM instead of SAM:
+//                 bwa mem | samtools view -B -T ref.fasta
+// TODO nf-core: Optional inputs are not currently supported by Nextflow. However, using an empty
+//               list (`[]`) instead of a file can be used to work around this issue.
+
```
Comment (maxulysse) [inline modules/nf-core/varlociraptor/call/main.nf:17]:
```suggestion
```

Comment (matthdsm) [reply modules/nf-core/varlociraptor/call/main.nf:17]: I'm just adding the skeleton from `nf-core modules create` 😄  Give me some time!

Comment (maxulysse) [reply modules/nf-core/varlociraptor/call/main.nf:17]: Yes, I figure that out, so I stopped after this first comment.
:-D

**Comment by maxulysse** [comment]
TODOs left in the meta.yml

**Pull request closed without merging**